testing/multi-env-test-engine/src/test/java/org/projectnessie/tools/compatibility/engine/TestMultiEnvTestEngine.java

```diff
@@ -46,8 +46,7 @@ void junitJupiter() {
                 .list()
                 .stream()
                 .map(e -> e.getTestDescriptor().getUniqueId()))
-        .isNotEmpty()
-        .allSatisfy(id -> assertThat(id.getEngineId()).hasValue(JupiterEngineDescriptor.ENGINE_ID));
```

Comment:
I think this test is not meaningful as a whole without this assertion. The handling of `PlainTest` is covered by the test below.

Reply:
There's a `.isEmpty()` below